src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/request/ApproveLicencesRequest.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.request
+
+import io.swagger.v3.oas.annotations.media.Schema
+
```

Comment:
Looking at the `getLicencesForApproval` call on the frontend, the only thing we require are the `filteredPrisons` as the other values are either null or can be plugged straight into the query to retrieve the relevant licences (see `LicenceRepository` and `LicenceService` changes).

Reply:
Yes, that makes sense